### DISCUSSION TITLE
vhost

### DISCUSSION THREAD

**Discussion by c4ir0** [projectdiscovery/httpx#796] [Q&A]
seems that the tgarget is supporting vhost, but what is the right way to make use of it please !!
` -vhost-input                  get a list of vhosts as input`

![13231](https://user-images.githubusercontent.com/108268587/194172677-c374e941-4799-43bb-9b52-3e2d8bd9f5bf.JPG)

**Comment by olearycrew** [comment]
@ehsandeep can you take a look at this question?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
